sound/hybrid_font.h

```diff
@@ -787,25 +811,38 @@ class HybridFont : public SaberBase, public Looper {
   void Loop() override {
     if (state_ == STATE_WAIT_FOR_ON) {
       if (!GetWavPlayerPlaying(&SFX_preon)) {
-	SaberBase::TurnOn(saved_location_);
-	return;
+        SaberBase::TurnOn(saved_location_);
+        return;
       }
     }
     if (check_postoff_) {
       if (!GetWavPlayerPlaying(&SFX_in) &&
-	  !GetWavPlayerPlaying(&SFX_poweroff) &&
-	  !GetWavPlayerPlaying(&SFX_pwroff)) {
-	check_postoff_ = false;
-	SaberBase::DoEffect(EFFECT_POSTOFF, saved_location_);
+          !GetWavPlayerPlaying(&SFX_poweroff) &&
+          !GetWavPlayerPlaying(&SFX_pwroff)) {
+        check_postoff_ = false;
+        SaberBase::DoEffect(EFFECT_POSTOFF, saved_location_);
       }
     }
   }
-  bool swinging_ = false;
+
+  void StopIdleSound() {
+#ifdef ENABLE_IDLE_SOUND
+    RefPtr<BufferedWavPlayer> idlePlayer = GetWavPlayerPlaying(&SFX_idle);
+    if (idlePlayer && idlePlayer->isPlaying()) {
```

Comment:
you don't need to check isPlaying(), GetWavPlayerPlaying already does that.

Reply:
I thought so. ok done.